test/load_test.c

```diff
@@ -0,0 +1,294 @@
+#include "../src/def/game_defs.h"
+#include "../src/def/game_history_defs.h"
+#include "../src/ent/game_history.h"
+#include "../src/impl/config.h"
+#include "../src/impl/load.h"
+#include "../src/util/io_util.h"
+#include "../src/util/string_util.h"
+#include "test_util.h"
+#include <assert.h>
+#include <stdbool.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+void test_cross_tables_integration(void) {
+  // Only run if MAGPIE_INTEGRATION_TESTS environment variable is set
+  if (!getenv("MAGPIE_INTEGRATION_TESTS")) {
+    printf("Skipping cross-tables integration test (set MAGPIE_INTEGRATION_TESTS=1 to enable)\n");
+    return;
+  }
+
+  GameHistory *game_history = game_history_create();
+  ErrorStack *error_stack = error_stack_create();
+
+  // Test with a known working cross-tables game URL
+  // Using a classic game that should be stable on cross-tables
+  Config *config = config_create_default_with_data_paths(error_stack, "data");
```

Comment:
Prefer to use config_create_or_die for all tests, that way you don't need to check for creation success.